org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/managers/GradleProjectImporterTest.java

```diff
@@ -158,6 +159,14 @@ public void testDisableGradleWrapper() throws Exception {
 		}
 	}
 
+	@Test
+	public void testGradleWrapper() throws Exception {
```

Comment:
needs to move to a WrapperValidatorTest class

Reply:
also need to add tests checking we can actually detect suspicious wrappers

Reply:
Fixed.